exec/csnex/Exec.cs

```diff
@@ -1165,6 +1168,9 @@ private void Invoke(Module m, int index)
         private int Loop(Int64 min_callstack_depth)
         {
             while (callstack.Count > min_callstack_depth && ip < module.Bytecode.code.Length && exit_code == 0) {
+                if (EnableTrace) {
+                    Console.Error.Write("mod {0} ip {1} ({2}) {3}\n", module.Name, ip, stack.Count, Disassembler.DisassembleInstruction(module.Bytecode, ip));
```

Comment:
Would it make more sense to use `WriteLine` here and omit the embedded `\n`?

Reply:
Fixed.